x-pack/plugins/observability/public/config/translations.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
```

Comment:
Please use `git mv` (move) instead of deleting and creating a new file, in order to preserve git history.

Reply:
Aaah this is an old habit. i am used to use IntelliJ and it does that git mv automatically if you drag and drop a file!